.travis.yml

```diff
@@ -60,6 +65,7 @@ branches:
 
 # setup requirements for running unit/integration/behat tests
 before_install:
+  - mysql -uroot -V
```

Comment:
can move this into bin/.travis/prepare_unittest.sh mysql section so we have it for debug

Reply:
It does:
```
Installing MariaDB version 10.3
mysql stop/waiting
Executing: /tmp/tmp.gHAPlJMavz/gpg.1.sh --recv-keys
--keyserver
hkp://keyserver.ubuntu.com:80
0xcbcb082a1bb943db
gpg: requesting key 1BB943DB from hkp server keyserver.ubuntu.com
gpg: key 1BB943DB: public key "MariaDB Package Signing Key <package-signing-key@mariadb.org>" imported
gpg: Total number processed: 1
gpg:               imported: 1
$ travis_apt_get_update
$ PACKAGES='mariadb-server mariadb-server-10.3'
$ if [[ $(lsb_release -cs) = 'precise' ]]; then PACKAGES="${PACKAGES} libmariadbclient-dev"; fi
16.13s$ sudo apt-get install -y -o Dpkg::Options::='--force-confnew' $PACKAGES
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  galera-3 mariadb-client-10.3 mariadb-client-core-10.3 mariadb-common
  mariadb-server-core-10.3
Suggested packages:
  mailx mariadb-test tinyca
The following packages will be REMOVED:
  mysql-client-5.6 mysql-client-core-5.6 mysql-server-5.6
  mysql-server-core-5.6
The following NEW packages will be installed:
  galera-3 mariadb-client-10.3 mariadb-client-core-10.3 mariadb-common
  mariadb-server mariadb-server-10.3 mariadb-server-core-10.3
0 upgraded, 7 newly installed, 4 to remove and 258 not upgraded.
Need to get 18.6 MB of archives.
After this operation, 12.8 MB of additional disk space will be used.
Get:1 http://nyc2.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu trusty/main amd64 mariadb-common all 1:10.3.10+maria~trusty [3,420 B]
Get:2 http://nyc2.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu trusty/main amd64 galera-3 amd64 25.3.24-trusty [7,257 kB]
Get:3 http://nyc2.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu trusty/main amd64 mariadb-client-core-10.3 amd64 1:10.3.10+maria~trusty [752 kB]
Get:4 http://nyc2.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu trusty/main amd64 mariadb-client-10.3 amd64 1:10.3.10+maria~trusty [1,132 kB]
Get:5 http://nyc2.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu trusty/main amd64 mariadb-server-core-10.3 amd64 1:10.3.10+maria~trusty [5,665 kB]
Get:6 http://nyc2.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu trusty/main amd64 mariadb-server-10.3 amd64 1:10.3.10+maria~trusty [3,782 kB]
Get:7 http://nyc2.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu trusty/main amd64 mariadb-server all 1:10.3.10+maria~trusty [3,064 B]
Fetched 18.6 MB in 4s (4,512 kB/s)
Preconfiguring packages ...
(Reading database ... 93695 files and directories currently installed.)
Removing mysql-server-5.6 (5.6.33-0ubuntu0.14.04.1) ...
Removing mysql-client-5.6 (5.6.33-0ubuntu0.14.04.1) ...
Removing mysql-client-core-5.6 (5.6.33-0ubuntu0.14.04.1) ...
Removing mysql-server-core-5.6 (5.6.33-0ubuntu0.14.04.1) ...
Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
Selecting previously unselected package mariadb-common.
(Reading database ... 93467 files and directories currently installed.)
Preparing to unpack .../mariadb-common_1%3a10.3.10+maria~trusty_all.deb ...
Unpacking mariadb-common (1:10.3.10+maria~trusty) ...
Selecting previously unselected package galera-3.
Preparing to unpack .../galera-3_25.3.24-trusty_amd64.deb ...
Unpacking galera-3 (25.3.24-trusty) ...
Selecting previously unselected package mariadb-client-core-10.3.
Preparing to unpack .../mariadb-client-core-10.3_1%3a10.3.10+maria~trusty_amd64.deb ...
Unpacking mariadb-client-core-10.3 (1:10.3.10+maria~trusty) ...
Selecting previously unselected package mariadb-client-10.3.
Preparing to unpack .../mariadb-client-10.3_1%3a10.3.10+maria~trusty_amd64.deb ...
Unpacking mariadb-client-10.3 (1:10.3.10+maria~trusty) ...
Selecting previously unselected package mariadb-server-core-10.3.
Preparing to unpack .../mariadb-server-core-10.3_1%3a10.3.10+maria~trusty_amd64.deb ...
Unpacking mariadb-server-core-10.3 (1:10.3.10+maria~trusty) ...
Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
Setting up mariadb-common (1:10.3.10+maria~trusty) ...
Selecting previously unselected package mariadb-server-10.3.
(Reading database ... 93611 files and directories currently installed.)
Preparing to unpack .../mariadb-server-10.3_1%3a10.3.10+maria~trusty_amd64.deb ...
/var/lib/mysql: found previous version 5.6
Unpacking mariadb-server-10.3 (1:10.3.10+maria~trusty) ...
Selecting previously unselected package mariadb-server.
Preparing to unpack .../mariadb-server_1%3a10.3.10+maria~trusty_all.deb ...
Unpacking mariadb-server (1:10.3.10+maria~trusty) ...
Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
Processing triggers for ureadahead (0.100.0-16) ...
Setting up galera-3 (25.3.24-trusty) ...
Setting up mariadb-client-core-10.3 (1:10.3.10+maria~trusty) ...
Setting up mariadb-client-10.3 (1:10.3.10+maria~trusty) ...
Setting up mariadb-server-core-10.3 (1:10.3.10+maria~trusty) ...
Setting up mariadb-server-10.3 (1:10.3.10+maria~trusty) ...
Configuration file '/etc/apparmor.d/usr.sbin.mysqld'
 ==> Deleted (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
 ==> Keeping old config file as default.
Installing new version of config file /etc/init.d/mysql ...
Installing new version of config file /etc/logrotate.d/mysql-server ...
Installing new version of config file /etc/mysql/debian-start ...
Installing new version of config file /etc/mysql/conf.d/mysqld_safe_syslog.cnf ...
mysql start/running, process 6267
Processing triggers for ureadahead (0.100.0-16) ...
Setting up mariadb-server (1:10.3.10+maria~trusty) ...
Starting MariaDB v10.3
0.02s$ sudo service mysql start
start: Job is already running: mysql
$ mysql --version
mysql  Ver 15.1 Distrib 10.3.10-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
git.checkout
3.10s$ git clone --depth=30 https://github.com/ezsystems/ezpublish-kernel.git ezsystems/ezpublish-kernel
Cloning into 'ezsystems/ezpublish-kernel'...
remote: Enumerating objects: 9242, done.
remote: Counting objects: 100% (9242/9242), done.
remote: Compressing objects: 100% (5763/5763), done.
remote: Total 9242 (delta 5168), reused 5273 (delta 3132), pack-reused 0
Receiving objects: 100% (9242/9242), 4.86 MiB | 8.06 MiB/s, done.
Resolving deltas: 100% (5168/5168), done.
$ cd ezsystems/ezpublish-kernel
0.53s$ git fetch origin +refs/pull/2468/merge:
remote: Enumerating objects: 60, done.
remote: Counting objects: 100% (34/34), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 10 (delta 6), reused 9 (delta 6), pack-reused 0
Unpacking objects: 100% (10/10), done.
From https://github.com/ezsystems/ezpublish-kernel
 * branch            refs/pull/2468/merge -> FETCH_HEAD
$ git checkout -qf FETCH_HEAD
Setting environment variables from .travis.yml
```

So IMHO I will skip this commit when rebasing before merge.